cookbook/security/user_checkers.rst

```diff
@@ -0,0 +1,221 @@
+.. index::
+single: Security; Creating and Enabling Custom User Checkers
+
+How to Create and Enable Custom User Checkers
+=============================================
+
+During the authentication of a user, additional checks might be required to verify
+if the identified user is allowed to log in. By defining custom user checkers, you
+can define per firewall which checkers should be used.
+
+.. versionadded:: 2.8
+Adding custom user checkers was introduced in Symfony 2.8.
+
+
+Creating a Custom User Checker
+------------------------------
+
+Symfony provides an interface that enables you to implement a custom user checker
```

Comment:
@javiereguiluz What about either of these? I personally think the latter would be sufficient.
```
Symfony provides an interface that enables you to create a custom 
user checker by implementing the required interface: `UserCheckerInterface`.
--- or ---
Symfony provides the `UserCheckerInterface` which enables you to 
create a custom user checker by implementing the interface.
```